### ISSUE TITLE
Provide installation script

### ISSUE THREAD

**Issue by deiruch** [i4Ds/Karabo-Pipeline#292]
Installation should be way easier. Something like `wget -O - https://ourserver.com/awesome-installation-and-update-script-for-karabo-but-find-a-better-name-first.sh | bash`. IMHO, installation requires too many steps as it is.

- [x] Get rid of the requirement to close the terminal during installation
- [x] Ensure the script works for updates as well
- [x] Not change the default solver to libmamba (`--solver=libmamba` (FYI: `--experimental-solver` is deprecated))
- [x] Consider using `--strict-channel-priority`

**Comment by deiruch** [comment]
Apart from reopening the console, and the question of installation path, the following script appears to work for initial setup.
```
wget https://repo.anaconda.com/miniconda/Miniconda3-py39_22.11.1-1-Linux-x86_64.sh
bash Miniconda3-py39_22.11.1-1-Linux-x86_64.sh -b -p $SCRATCH/miniconda3
export PATH="/home/username/miniconda3/bin:$PATH"
conda install -y -n base conda-libmamba-solver
conda config --set solver libmamba
conda update -y -n base -c defaults conda
conda create -y -n karabo-env python=3.9
conda init bash

>>> close & reopen terminal <<<

export PATH="$SCRATCH/miniconda3/bin:$PATH"
conda activate karabo-env
conda install -y -c i4ds -c conda-forge -c nvidia/label/cuda-11.7.0 karabo-pipeline
```

- [ ] Get rid of the requirement to close the terminal manually
- [ ] Ensure the script works for updates as well

**Comment by Lukas113** [comment]
@deiruch do you have something different in mind that what's already written in `doc.src.installation_user.md`?

PS: I don't recommend installing conda in scratch, because it's not safe from removal.

**Comment by deiruch** [comment]
- Yes, most of this is done.
- SCRATCH is way faster to install, which is why I suggested it. But that's obviously only reasonable for quick experiments. I think you're right that we shouldn't suggest users use SCRATCH. Do you know how slow the installation in HOME is on CSCS?
- What do you think about `--strict-channel-priority`? Not in favour? I'd like to add it to our scripts.

**Comment by Lukas113** [comment]
I can't say it explicitly, but the installation in my home directory and the following installation of Karabo took about 5 min for me, where the installation of Karabo took most of the time. I think it's about 1 min or so. Installing it in HOME doesn't really come with costs, except it's automatically active once you added the initialization in your .bashrc (which is my preferred way).

I don't have a strong opinion towards --strict-channel-priority.

**Comment by deiruch** [comment]
Closed with PR #539
